Resources/Textures/Objects/Materials/sheets.rsi/meta.json

```diff
@@ -165,6 +165,46 @@
             "name": "plasteel",
             "directions": 1
         },
+        {
+        	"name": "rglass",
+        	"directions": 1
+        },
+        {
+        	"name": "rglass-inhand-left",
+        	"directions": 4,
+        	"delays": [
+                [
+                    1
+                ],
+                [
+                    1
+                ],
+                [
+                    1
+                ],
+                [
+                    1
+                ]
```

Comment:
Don't need delays if it's just 1

Also is the licence for these the same as the other states?

Reply:
No idea ask swept